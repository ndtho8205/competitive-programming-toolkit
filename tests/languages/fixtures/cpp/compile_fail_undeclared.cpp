#include <iostream>

int main() {
  a = 0;
}
