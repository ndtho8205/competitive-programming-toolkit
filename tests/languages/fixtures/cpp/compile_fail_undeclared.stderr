compile_fail_undeclared.cpp: In function ‘int main()’:
compile_fail_undeclared.cpp:4:3: error: ‘a’ was not declared in this scope
   a = 0;
   ^
